flink-streaming-java/src/test/java/org/apache/flink/streaming/api/environment/StreamExecutionEnvironmentTest.java

```diff
@@ -446,16 +446,20 @@ void testBufferTimeoutEnabled() {
     void testBufferTimeoutDisabled() {
         Configuration config = new Configuration();
         config.set(ExecutionOptions.BUFFER_TIMEOUT_ENABLED, false);
-        StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
+        StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment(config);
```

Comment:
Not sure why passing the config in here? It will be used in `env.configure()` right after.